source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.TestCore/InstantHelper.cs

```diff
@@ -48,5 +48,21 @@ public static Instant GetTodayPlusDaysAtMidnightUtc(int noOfDaysToAdd)
         {
             return Instant.FromDateTimeUtc(DateTime.Now.AddDays(noOfDaysToAdd).Date.ToUniversalTime());
         }
+
+        public static Instant GetThisMonthPlusMonthsAtMidnightUtc(int noOfMonthsToAdd)
```

Comment:
Perhaps something like this 
```suggestion
        public static Instant GetFirstDayOfThisMonthPlusMonthsAtMidnightUtc(int noOfMonthsToAdd)
```